controllers/deployment_controller.go

```diff
@@ -46,12 +51,17 @@ func (r *DeploymentReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	deployment := new(csibaremetalv1.Deployment)
 	err := r.Get(ctx, client.ObjectKey{Name: req.Name, Namespace: req.Namespace}, deployment)
 	if err != nil {
+		if errors.IsNotFound(err) {
+			log.Info("Custom resource is not found")
```

Comment:
Why is this log message with Info level?

Reply:
This message appear if user deletes CR (or helm release now) from cluster and I think it isn't an error. Or should I change it to Error level?

Reply:
Lets have `warning` at least.

Reply:
We use `github.com/go-logr/logr` and it has no Warn level, only Info and Error. It would be better to change logger or can I make issue for it?

Reply:
Please add todo and create issue for log library change.

Reply:
done